auth/src/main/java/io/grpc/auth/GoogleAuthLibraryCallCredentials.java

```diff
@@ -214,6 +235,19 @@ static JwtHelper createJwtHelperOrNull(ClassLoader loader) {
     return null;
   }
 
+  @Nullable
+  private static Class<? extends Credentials> loadGoogleCredentialsClass() {
+    Class<?> rawGoogleCredentialsClass;
+    try {
+      // Can't use a loader as it disables ProGuard's reference detection and would fail to rename
+      // this reference. Unfortunately this will initialize the class.
+      rawGoogleCredentialsClass = Class.forName("com.google.auth.oauth2.GoogleCredentials");
+    } catch (ClassNotFoundException ex) {
+      return null;
+    }
+    return rawGoogleCredentialsClass.asSubclass(Credentials.class);
```

Comment:
Credentials isn't a subclass is it?

Reply:
I don't understand your question :)

Reply:
You are calling asSubclass, but infact Credentials is a super class

Reply:
Javadoc says "Casts this Class object to represent a subclass of the class represented by the specified class object." So it returns "subclass of Credentials".